flink-runtime/src/test/java/org/apache/flink/runtime/jobmanager/JobManagerProcessUtilsTest.java

```diff
@@ -69,6 +69,20 @@ public void testConfigOffHeapMemory() {
 			jobManagerProcessSpec -> assertThat(jobManagerProcessSpec.getJvmDirectMemorySize(), is(offHeapMemory)));
 	}
 
+	@Test
+	public void testConfigJvmHeapAndOffHeapMemory() {
```

Comment:
This test tests 2 things: heap and off-heap settings.
Also, I think it is redundant as it is duplicated with `testConfigJvmHeapMemory` and `testConfigOffHeapMemory`

Reply:
@azagrebin I previously want to add test for the case that the off-heap size is less than jobstore cache size. I have removed this test method.